src/renderer/atlas/AtlasEngine.h

```diff
@@ -359,7 +359,9 @@ namespace Microsoft::Console::Render
 
             bool is_inline() const noexcept
             {
-                return (__builtin_bit_cast(uintptr_t, allocated) & 1) != 0;
+                // VSO-1430353: __builtin_bitcast crashes the compiler under /permissive-.

```

Comment:
LOL.

Reply:
...lol.

Reply:
```suggestion
                // VSO-1430353: __builtin_bitcast crashes the compiler under /permissive-. (BODGY)
```

working around compiler bugs is a BODGE